File: C++/1008.cpp
#include <iostream>
using namespace std;

int main()
{
    int a, b;
    float c;
    cin >> a >> b >> c;
    cout << "NUMBER = " << a << endl;
    cout.precision(2);
    cout << fixed << "SALARY = U$ " << (b * c) << endl;
    return 0;
}

File: Dart/1008.dart
import 'dart:io';

void main() {
  int num = int.parse(stdin.readLineSync());
  int salary = int.parse(stdin.readLineSync());
  double bonus = double.parse(stdin.readLineSync());
  double total = salary * bonus;

  print("NUMBER = $num");
  print("SALARY = U\$ " + total.toStringAsFixed(2));
}
